website/docs/sdk-reference/unreal.md

```diff
@@ -0,0 +1,712 @@
+- you've completed the [Visual Studio](https://docs.unrealengine.com/5.2/en-US/setting-up-visual-studio-development-environment-for-cplusplus-projects-in-unreal-engine/) or [Visual Studio Code](https://docs.unrealengine.com/5.2/en-US/setting-up-visual-studio-code-for-unreal-engine/) setup
+
+Run the following commands in the root folder of your project (where the `.uproject` file is located):
+
+```cmd
+mkdir -p Plugins
+cd Plugins
+git clone https://github.com/configcat/unreal-engine-sdk ConfigCat
+```
+
+### 2. Enable the ConfigCat plugin in your project
+
+<img className="unreal-enable-plugin zoomable" src="/docs/assets/unreal/enable-plugin.png" alt="Unreal Engine enable plugin" />
+
+Navigate to `Edit -> Plugins` and perform the following steps:
+1. Find the `Config Cat` plugin and **tick** the enable checkbox.
+1. Press `Restart` to load up the editor.
+
+Note: if you are using a locally built plugin, you will need to rebuild from source manually.
+
+### 3. Set up the _ConfigCat_ settings with your _SDK Key_
+
+You can configure all ConfigCat related settings inside the `ProjectSettings -> Feature Flags -> ConfigCat`.
+
+<img className="unreal-plugin-settings zoomable" src="/docs/assets/unreal/plugin-settings.png" alt="Unreal Engine plugin settings" />
+
+| Properties                    | Description                                                                                                                                                                                                                                                                                         |
+| ----------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `Sdk Key`                     | SDK Key to access your feature flags and configurations. Get it from ConfigCat Dashboard.                                                                                                                                                                                                           |
+| `Base Url`                    | Optional, sets the CDN base url (forward proxy, dedicated subscription) from where the sdk will download the config JSON.                                                                                                                                                                        |
+| `Data Governance`             | Optional, defaults to `Global`. Describes the location of your feature flag and setting data within the ConfigCat CDN. This parameter needs to be in sync with your Data Governance preferences. [More about Data Governance](advanced/data-governance.md). Available options: `Global`, `EuOnly`.  |
```

Comment:
```suggestion
| `Data Governance`             | Optional, defaults to `Global`. Describes the location of your feature flag and setting data within the ConfigCat CDN. This parameter needs to be in sync with your Data Governance preferences. [More about Data Governance](/advanced/data-governance). Available options: `Global`, `EuOnly`.  |
```
The internal link format changed a bit, and the correct ones should start with a / and not end with .md